### DISCUSSION TITLE
Does Workflows view now show archived workflows?

### DISCUSSION THREAD

**Discussion by driv** [argoproj/argo-workflows#12545] [Q&A] [answered]
After an upgrade the Workflow view is showing really old Workflows that are archived. Is this expected?
Is there any way of either setting a default time from or just showing non-archived Workflows?

**Comment by agilgur5** [comment, marked as answer]
> After an upgrade the Workflow view is showing really old Workflows that are archived. Is this expected?

Yes, [per the release notes](https://blog.argoproj.io/whats-new-in-argo-workflows-v3-5-f260e8603ca6), v3.5 merged the Archive UI with the regular live Workflows List UI.

> Is there any way of either setting a default time from or just showing non-archived Workflows?

A patch is being worked on to support not showing Archived Workflows per https://github.com/argoproj/argo-workflows/issues/12025#issuecomment-1865458377. **EDIT**: See dedicated issue: https://github.com/argoproj/argo-workflows/issues/13171

The date filter in the UI is currently [not connected to `localStorage`](https://github.com/argoproj/argo-workflows/blob/a15587755e250fbbe3b9538b1c15ae581a108f0c/ui/src/app/workflows/components/workflows-list/workflows-list.tsx#L124), but could open a feature request to make that possible. Then the previously set date filter will persist throughout UI usage

**Reply by agilgur5** [reply]
Following up here:

> Yes, [per the release notes](https://blog.argoproj.io/whats-new-in-argo-workflows-v3-5-f260e8603ca6), v3.5 merged the Archive UI with the regular live Workflows List UI.

I wrote up a PR to add this to the upgrading notes and the new version modal in the UI as well: #13371



> A patch is being worked on to support not showing Archived Workflows per [#12025 (comment)](https://github.com/argoproj/argo-workflows/issues/12025#issuecomment-1865458377).

This has been split out to #13171 for tracking purposes and is still slated for a 3.5.x patch enabling it
